### PR TITLE
Encourage creation of aiohttp public objects inside a coroutine

### PR DESCRIPTION
**Pull request by ZYunH**
Replace asyncio.get_event_loop() with get_running_loop()

Nope

Nope

#3331 

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by zhangyunhao116** [comment]
May be i should rebase those two commits for test ? @asvetlov

**Comment by gyermolenko** [comment]
@ZYunH towncrier fails on python 3.6, `asyncio.get_event_loop()` was added in 3.7. Maybe that's the case.
Also you can run `flake8 aiohttp` to resolve build failure for linter.

**Comment by zhangyunhao116** [comment]
@gyermolenko thx! i will fix this case

File: aiohttp/base_protocol.py [hunk 1]
```diff
@@ -10,7 +11,7 @@ class BaseProtocol(asyncio.Protocol):
 
     def __init__(self, loop: Optional[asyncio.AbstractEventLoop]=None) -> None:
         if loop is None:
-            self._loop = asyncio.get_event_loop()
+            self._loop = get_running_loop()
```
[the remark this answers is not in the document]
Comment (zhangyunhao116) [reply aiohttp/base_protocol.py]: Got you.I will do this.
BTW,is there any plan to use asyncio.get_running_loop()  which is a new api in python3.7 ?

[the remark this answers is not in the document]
Comment (asvetlov) [reply aiohttp/base_protocol.py]: Eventually yes, but we should support Python 3.5 and 3.6 for a long time

**Comment by zhangyunhao116** [comment]
I think it is simple at first, but it is wrong.There is still a lot of work to be done for test infra.Is there any easy way to make the test infra(more than 200 DeprecationWarning in test suites)? 
And also, get event loop form another module make something wrong, i guess it create loop from other threads(i.e test.connector.test_ctor_loop()) or might I got that wrong. @asvetlov

**Comment by asvetlov** [comment]
Test suite disables default event loop, this is why you see ` RuntimeError: There is no current event loop in thread 'MainThread'`.

In general converting `def test_xxx()` into `async def test_xxx()` can solve the issue.
Sometimes it is not easy.

Please fix trivial cases and let me know, I'll pick up the PR and finish it if needed.

**Comment by zhangyunhao116** [comment]
@asvetlov Thanks for reply! Now these commits only do
1) Add get_running_loop() in helpers.py
2) Replace asyncio.get_event_loop() with get_running_loop() in aiottp/*.py
except two files:
abc.py -> Replace will cause circular import error
web.py -> run_app() is not coroutine function, and will always make warning

No unit tests for the changes

**Comment by asvetlov** [comment]
1. #3336 should help with circular imports.
2. You are right, `run_app()` should keep using `get_event_loop()`

**Comment by zhangyunhao116** [comment]
Now i can't access this PR, because previous fork is deleted for next fix.Pick up this PR if you need.
Thanks for all the work, and very nice product!

**Comment by asvetlov** [comment]
Done in a separate commit

**Pull request closed without merging**